blinkpy/camera.py

```diff
@@ -98,21 +99,20 @@ def arm(self):
         """Return arm status of camera."""
         return self.motion_enabled
 
-    @arm.setter
-    def arm(self, value):
+    async def async_arm(self, value):
         """Set camera arm status."""
         if value:
-            return api.request_motion_detection_enable(
+            return await api.request_motion_detection_enable(
                 self.sync.blink, self.network_id, self.camera_id
             )
-        return api.request_motion_detection_disable(
+        return await api.request_motion_detection_disable(
             self.sync.blink, self.network_id, self.camera_id
         )
 
     @property
-    def night_vision(self):
+    async def night_vision(self):
```

Comment:
I think there is something fundamentally wrong with this function.
I was told by someone over in the HA project that properties should not have side effects and also should just be returning stored values.  This value should probably be updated with the update functions then this wouldn't be `async` and would just  `return {key: res.get(key) for key in nv_keys}` where the values were stored in a self.xxx